Library/Formula/open-jtalk.rb

```diff
@@ -0,0 +1,51 @@
+class OpenJtalk < Formula
+  homepage "http://open-jtalk.sourceforge.net/"
+  url "https://downloads.sourceforge.net/project/open-jtalk/Open%20JTalk/open_jtalk-1.08/open_jtalk-1.08.tar.gz"
+  sha1 "34749abce5f8263ebbe9843b92407f4f0a742c66"
+
+  depends_on "gcc" => :build
```

Comment:
Need to use `fails_with` instead of a `gcc` dependency and we'll probably not accept this until it compiles with Clang.